packages/@vuepress/core/lib/app/app.js

```diff
@@ -62,15 +62,25 @@ export function createApp (isServer) {
     base: siteData.base,
     mode: 'history',
     fallback: false,
-    routes
+    routes,
+    scrollBehavior (to, from, savedPosition) {
```

Comment:
What about cases when `to.hash` is set?

Reply:
That interferes with `plugin-active-header-links` unfortunately. It causes the website to jump around when the anchor is being pushed.

Reply:
Alright perfect, then i guess in the old implementation it was just to disable scroll behavior via some flag.

Reply:
Ah that would make sense here, too! Have to check that out later.